src/view/drag-handle/util/should-allow-dragging-from-target.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import type { Props } from '../drag-handle-types';
+
+export const interactiveTagNames: string[] = [
+  'input',
+  'textarea',
+  'select',
+  'option',
+  'button',
+  'contenteditable',
```

Comment:
`contenteditable` isn't a tag name, it's an attribute that can be applied to elements.